lib/perl/Genome/Config/Command/Get.pm

```diff
@@ -0,0 +1,31 @@
+package Genome::Config::Command::Get;
+
+use strict;
+use warnings;
+
+use Genome qw();
+use Genome::Config qw();
+
+class Genome::Config::Command::Get {
+    is => 'Command::V2',
+    has => [
+        keys => {
+            is => 'Text',
+            is_many => 1,
+            is_optional => 1,
+            shell_args_position => 1,
+        },
+    ],
+};
+
+sub execute {
+    my $self = shift;
+    my @keys = $self->keys || Genome::Config::all_keys();
```

Comment:
Actually, I guess I'm just using it wrong.  It should probably return an arrayref.